bin/stratis

```diff
@@ -22,6 +22,7 @@ try:
     import sys
 
     from stratis_cli import run
+    from stratis_cli._errors import StratisCliEnvironmentError
```

Comment:
Export ```StratisCliEnvironmentError``` in ```stratis_cli/__init__.py```, and import it directly from stratis_cli here. You should do the same in ```test_timeout.py```.